### PR TITLE
Implement disableAutomaticCheckForUpdate

### PR DESCRIPTION
**Pull request by annakocheshkova**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [X] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [X] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
image

[AB#75292](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/75292)

### CODE REVIEW

File: AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/AppCenterLoader.java [hunk 1]
```diff
@@ -107,6 +108,9 @@ public boolean onCreate() {
             if (updateTrackValue != null) {
                 Distribute.setUpdateTrack(Integer.parseInt(updateTrackValue));
             }
+            if (isTrueValue(getStringResource(DISTRIBUTE_DISABLE_CHECK_FOR_UPDATE_KEY))) {
+                //Distribute.disableAutomaticCheckForUpdates();
```
Comment (jaeklim) [inline AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/AppCenterLoader.java]: No `s`.

File: Assets/AppCenter/Plugins/iOS/Core/AppCenterStarter.original [hunk 2]
```diff
@@ -107,6 +107,10 @@ static const int kMSUpdateTrack = 1 /*UPDATE_TRACK*/;
 #endif // APPCENTER_UNITY_USE_CUSTOM_INSTALL_URL
   [classes addObject:MSDistribute.class];
 
+#ifdef APPCENTER_DISTRIBUTE_DISABLE_CHECK_FOR_UPDATE
+  [MSDistribute disableAutomaticCheckForUpdates];
```
Comment (jaeklim) [inline Assets/AppCenter/Plugins/iOS/Core/AppCenterStarter.original]: No `s`.

**Comment by jaeklim** [comment]
Looks good to me so far.

File: AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/AppCenterLoader.java [hunk 3]
```diff
@@ -107,6 +108,10 @@ public boolean onCreate() {
             if (updateTrackValue != null) {
                 Distribute.setUpdateTrack(Integer.parseInt(updateTrackValue));
             }
+            if (isTrueValue(getStringResource(DISTRIBUTE_DISABLE_CHECK_FOR_UPDATE_KEY))) {
+                AppCenterLog.debug(AppCenterLog.LOG_TAG, "TEST_1243");
```
Comment (MatkovIvan) [inline AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/AppCenterLoader.java]: Don't forget to remove it 😅

**Review by jaeklim — APPROVED** [review]

**Review by AnastasiaKubova — APPROVED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**